src/Hl7.Fhir.Core/Rest/HttpToEntryExtensions.cs

```diff
@@ -46,7 +46,13 @@ internal static Bundle.EntryComponent ToBundleEntry(this HttpWebResponse respons
 
 #if NETSTANDARD1_1
             if (!String.IsNullOrEmpty(response.Headers[HttpUtil.LASTMODIFIED]))
-                    result.Response.LastModified = DateTimeOffset.Parse(response.Headers[HttpUtil.LASTMODIFIED]);
+            {
+                DateTimeOffset dateTimeOffset = new DateTimeOffset();
+                bool success = DateTimeOffset.TryParse(response.Headers[HttpUtil.LASTMODIFIED], out dateTimeOffset);
+                if (!success)
+                    throw new FormatException($"String \"{response.Headers[HttpUtil.LASTMODIFIED]}\" was not recognized as a valid DateTime");
```

Comment:
The error message should mention that this error was found while parsing the LastModified header, otherwise we're still not providing more context ;-)

Reply:
Such a trivial issue, I fixed it for you.